code/modules/integrated_electronics/subtypes/weaponized.dm

```diff
@@ -138,9 +138,9 @@
 	A.preparePixelProjectile(target, src)
 	A.fire()
 	if(ismob(loc.loc))
-		installed_gun.shoot_live_shot(loc.loc)
+		installed_gun.shoot_live_shot(loc.loc, FALSE, null, TRUE, 0)
```

Comment:
These are the default args for the proc; this won't fix runtimes, as this change compiles to the exact same bytecode